specs/ledger/hs/src/Ledger/Update.hs

```diff
@@ -388,7 +391,7 @@ instance STS ADDVOTE where
               [(pid, vks) | vks <- Set.toList . Map.findWithDefault Set.empty vk $ invertMap dms ]
         Set.member pid rups ?! NoUpdateProposal pid
         Core.verify vk pid (vote ^. vSig) ?! AVSigDoesNotVerify
-        return $! Set.union vts vtsPid
+        return $! Core.PairSet $ Set.union (Core.unPairSet vts) vtsPid
```

Comment:
Does it make sense to define a semigroup instance for pairset? If so I can make a small issue for this.

Reply:
Yes, please go ahead!

Reply:
https://github.com/input-output-hk/fm-ledger-rules/issues/262